### PR TITLE
Phoebemirman/ch9433/add finalize button for markets awaiting

### PR DESCRIPTION
**Pull request by phoebemirman**
[Clubhouse Story](https://app.clubhouse.io/augur/story/9433)

## Submitter checklist
- [ ] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [ ] Steps to reproduce
  - [ ] Screenshot (If applicable)

## Reviewer Checklist
- [ ] Test/lint pass

### CODE REVIEW

**Comment by adrake33** [comment]
This pull request **introduces 1 alert** when merging 2e6b0e5890558d56f73da2648fda33eaaa041ab6 into 6dcf9cce34457002d1ad1ed4ac6e66e437e5ebb0 - [view on lgtm.com](https://lgtm.com/projects/g/AugurProject/augur/rev/pr-8a8cc46da3ca95d9720df58033c71b76e987e747)

**new alerts:**

* 1 for Useless conditional

---

*Comment posted by [lgtm.com](https://lgtm.com)*

File: src/modules/market/components/market-portfolio-card/market-portfolio-card-footer.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import Styles from 'modules/market/components/market-portfolio-card/market-portfolio-card.styles'
+import { constants } from 'services/augurjs'
+import moment from 'moment'
+
+const MarketPortfolioCardFooter = (p) => {
+  const WrappedGraph = TimeRemainingIndicatorWrapper(SingleSlicePieGraph)
+  const showTimestamp = p.linkType === TYPE_CLAIM_PROCEEDS && !p.isClaimable
+  const finalDate = new Date(p.finalizationTime*1000)
+  const startTime = moment(p.finalDate).subtract(constants.CONTRACT_INTERVAL.CLAIM_PROCEEDS_WAIT_TIME, 'seconds').toDate()
+  return (
```
Comment (bthaile) [inline src/modules/market/components/market-portfolio-card/market-portfolio-card-footer.jsx]: I think we want to show the time and wrappedGraph if the market is finalized

File: src/modules/market/components/market-portfolio-card/market-portfolio-card-footer.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                <span className={Styles['MarketCard__proceeds-clock']}>
+                  <WrappedGraph startDate={startTime} endTime={finalDate} currentTimestamp={p.currentTimestamp} />
+                </span>
+              </div>
+            }
+            <button
+              className={classNames(Styles['MarketCard__action-footer'], Styles['MarketCard__action-footer-light'])}
+              onClick={p.buttonAction}
+              disabled={p.linkType === TYPE_CLAIM_PROCEEDS ? !p.isClaimable : false}
+            >
```
Comment (bthaile) [inline src/modules/market/components/market-portfolio-card/market-portfolio-card-footer.jsx]: we want the button to be disabled if the current augur timestamp is beyond the finalizationTime + CLAIM_PROCEEDS_WAIT_TIME.

File: src/modules/market/components/market-portfolio-card/market-portfolio-card-footer.jsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+          className={classNames(
+            Styles['MarketCard__headingcontainer-footer'],
+            Styles['MarketCard__headingcontainer-footer-light'],
+          )}
+        >
+          {p.linkType === TYPE_CLAIM_PROCEEDS &&
+            <div>
+              <span className={Styles['MarketCard__light-text']}>Outstanding Returns</span>
+              <span className={Styles['MarketCard__heavy-text']}>{p.outstandingReturns}</span>
+            </div>
```
Comment (bthaile) [inline src/modules/market/components/market-portfolio-card/market-portfolio-card-footer.jsx]: Need to format p.outstandingReturns use formatEther in format-number

**Pull request merged**